File: collie/src/main/java/com/snail/collie/CollieListener.java
package com.snail.collie;

import com.snail.collie.battery.BatteryStatsTracker;
import com.snail.collie.fps.ITrackFpsListener;
import com.snail.collie.mem.MemoryLeakTrack;
import com.snail.collie.startup.LauncherTracker;
import com.snail.kotlin.trafficstats.ITrackTrafficStatsListener;

public interface CollieListener extends LauncherTracker.ILaunchTrackListener, ITrackFpsListener, MemoryLeakTrack.ITrackMemoryListener, ITrackTrafficStatsListener, BatteryStatsTracker.IBatteryListener {

}

File: collie/src/main/java/com/snail/collie/CollieListener.kt
package com.snail.collie

import com.snail.collie.battery.BatteryStatsTracker.IBatteryListener
import com.snail.collie.mem.MemoryLeakTrack.ITrackMemoryListener
import com.snail.collie.fps.ITrackFpsListener
import com.snail.collie.startup.LauncherTracker
import com.snail.collie.trafficstats.ITrackTrafficStatsListener

interface CollieListener : LauncherTracker.ILaunchTrackListener, ITrackFpsListener, ITrackMemoryListener, ITrackTrafficStatsListener, IBatteryListener
